### PR TITLE
[Done] reorder longlist item

### PR DESCRIPTION
**Pull request by bhavinsen**

### CODE REVIEW

File: src/store/item.ts [hunk 1]
```diff
@@ -30,6 +30,11 @@ export const useItemStore = defineStore("item", {
       return this.saveData(podcastname, docname)
     },
 
+    async updatesoltItem(state: State,item: any, podcastname: string, docname: string) {
```
Comment (chrisns) [inline src/store/item.ts:33]: `updatesoltItem`
not sure what this is meant to mean?

File: src/pages/Home.vue [hunk 2]
```diff
@@ -80,12 +80,34 @@ const dragged = (x: number, y: number, item: Item) => {
     <string>// @ts-ignore
     document.elementFromPoint(x, y)?.closest("section")?.attributes["slotno"]?.value,
   )
+
+  if(slot && slot == item.slot){
```
Comment (chrisns) [inline src/pages/Home.vue:84]:
```suggestion
  if(slot && slot === item.slot){
```

**Review by chrisns — CHANGES_REQUESTED** [review]
could also do with running `eslint --fix .` to catch the obvious formatting things

**Comment by chrisns** [comment]
You'll also note the build failures:
https://github.com/chrisns/smartseven/runs/6227045968?check_suite_focus=true#step:6:8

File: src/store/item.ts (hunk 1 above)
Comment (bhavinsen) [reply src/store/item.ts]: After reordering items then we need to update state value so I have created this function (`updatesoltItem`)

Comment (chrisns) [reply src/store/item.ts]: understood, i'm just not sure what you meant to write, presumably `update slot Item` but misspelled `slot` dnd didn't capitalise it.
If thats right, suggest renaming it to `updateSlotItem`

**Pull request merged**